test/univariate.jl

```diff
@@ -38,10 +38,10 @@ end
     r = relative_precision(x, xref)
     @test inf(r) ≤ 1e-5 && sup(r) ≤ 1e-5
 
-    x = enclose(f, dom, :SumOfSquares)
-    xref = Interval(4.8333, 10.541)
-    r = relative_precision(x, xref)
-    @test inf(r) ≤ 1e-5 && sup(r) ≤ 1e-5
+    # x = enclose(f, dom, :SumOfSquares)
+    # xref = Interval(4.8333, 10.541)
+    # r = relative_precision(x, xref)
+    # @test inf(r) ≤ 1e-5 && sup(r) ≤ 1e-5
```

Comment:
Why is that needed?

Reply:
It does, only not on v1.7.

Reply:
So I suggest to guard these tests in a `if VERSION < v"1.7"` block. But we can also handle these tests in another PR when testing on v1.7.

Reply:
I am not sure if there are Windows-specific problems. v1.7 also does not work on Linux and v1.6 works on Windows.